### PR TITLE
local node setup page first commit

### PR DESCRIPTION
**Pull request by yhaspel**
Screen Shot 2019-03-31 at 11 39 37 AM

Local Node Setup page:
Left Pane (setup mode only)
Right pane (setup, progress, overview modes)

### CODE REVIEW

File: app/screens/localNode/LocalNode.js [hunk 1]
```diff
@@ -20,35 +25,97 @@ const Header = styled.span`
...
-const RightPane = styled.div`
+const RightPaneWrapper = styled.div`
   flex: 1;
   background-color: white;
   padding: 30px;
   border: 1px solid ${smColors.borderGray};
 `;
 
-class LocalNode extends Component<{}> {
+type Mode = 'setup' | 'progress' | 'overview';
```
Comment (IlyaVi) [inline app/screens/localNode/LocalNode.js]: move this to enums

File: app/screens/localNode/LocalNode.js [hunk 2]
```diff
@@ -20,35 +25,97 @@ const Header = styled.span`
...
+    const { mode } = this.state;
     const header = `Local Node${mode !== 'overview' ? ' Setup' : ''}`;
     return (
       <Container>
         <Header>{header}</Header>
         <BodyWrapper>
-          <LeftPane>Left Pane</LeftPane>
-          <RightPane>Right Pane</RightPane>
+          <LeftPaneWrapper>{this.renderLeftPane(mode)}</LeftPaneWrapper>
+          <RightPaneWrapper>
```
Comment (IlyaVi) [inline app/screens/localNode/LocalNode.js]: use same logic as for LeftPane

File: app/screens/localNode/LocalNode.js (hunk 1 above)
Comment (yhaspel) [reply app/screens/localNode/LocalNode.js]: np

File: app/screens/localNode/LocalNode.js (hunk 2 above)
Comment (yhaspel) [reply app/screens/localNode/LocalNode.js]: consolidated into one component

**Pull request merged**